### DISCUSSION TITLE
Is there any way to set the icon in the upper left corner of the installation interface?

### DISCUSSION THREAD

**Discussion by BingoZha** [wixtoolset/issues#8566] [Questions] [answered]
### WiX Version

5.0.0

### .NET or MSBuild or Visual Studio Version

.Net 8.0

### HeatWave Version

1.0.4

### Windows Version

Win11 23H2

### Repro Repo

https://github.com/wixtoolset/wix

### Repro Steps

1. Generate a msi file.
2. Build a .exe package.

### Actual Result

I can't find any property to set this icon:
![1718353825509](https://github.com/wixtoolset/issues/assets/13460673/61802cee-7915-4bea-b502-fc232b67cfe7)

### Acknowledgements

- [X] I acknowledge that this is a fully completed bug report. It is not a question or attempt to get help debugging my issue (because those should be sent to Discussions).

**Comment by campersau** [comment, marked as answer]
See https://github.com/wixtoolset/issues/issues/8104#issuecomment-2074635376 for a workaround.

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [GitHub discussions area][gh]. You might also consider asking your question on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[so]: https://stackoverflow.com/questions/tagged/wix
